src/treq/test/test_content.py

```diff
@@ -207,14 +207,16 @@ def test_content_application_json_default_encoding(self):
 
     def test_text_content_unicode_headers(self):
         """
-        Header parsing is robust against unicode header names and values.
+        Parsing of the Content-Type header is robust in the face of
+        Unicode gunk in the header value, which is ignored.
```

Comment:
I am not familiar with the code from here... so feel free to ignore this comment.

I don't understand the docstring for this test...

I mean, I understand what it wanted to do, but I don't see how this is put into practice in the test code itself.

```suggestion
        Unicode junk in the header value, which is ignored.
```

Reply:
It can also be a unicode hunk (in this case meant as a byte sequence \xe1\x9b\x83 - which is the byte representation of the utf-8 code point of the given character).

All those bytes are invalid for http header value.

Reply:
Exactly &mdash; "ᛃ" is the "gunk".